app/src/main/res/layout/activity_redirect.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".RedirectActivity">
+
+
+    <LinearLayout
+        android:layout_width="match_parent"
```

Comment:
Moim zdaniem zbędne - za pomocą ConstraintLayout prawdopodobnie da się osiągnąć ten sam wygląd i unikamy wtedy kolejnego layoutu. Po za tym, mając tu LinearLayout jeślibyśmy chcieli umieścić przycisk na samym dole, to albo to niemożliwe albo wymaga strasznego kombinowania (patrz notkę niżej)